styles.css

```diff
@@ -67,3 +66,241 @@ ul {
+  font-family: 'Inter', sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 16px;
+  color: #091e42;
+  margin-top: 12px;
+  line-height: 24px;
+}
+
+.Multi-Post-ul {
+  width: 100%;
+  margin-bottom: 15px;
+}
+
+.Multi-Post-li {
+  border: 1px solid #8993a4;
+  padding: 9px 12px;
+  margin-right: 10px;
+  font-family: 'Inter', sans-serif;
+  font-weight: 500;
+}
+
+.Multi-Post-project {
+  align-items: center;
+  gap: 10px;
+  width: 124px;
+  height: 48px;
+  background: #ff6b00;
+  color: #fff;
+  border-style: none;
+}
+
+/* ...........work section.......... */
+.work-container {
+  width: 90%;
+  margin: auto;
+  background-color: #fff;
+  padding-bottom: 120px;
+}
+
+.art-container {
+  background-image: url("/Images/Img\ Placeholder\ \(1\).png");
+}
+
+.profesional-Art-heading {
+  font-family: 'Crete Round', sans-serif;
+  font-weight: 400;
+  font-size: 32px;
+  width: 295px;
+  height: 81px;
+  line-height: 44px;
+  color: #fff;
+  padding: 80px 0 0 10px;
+}
+
+.profesional-Art-content {
+  font-size: 16px;
+  line-height: 24px;
+  font-weight: 400px;
+  font-family: 'Inter', sans-serif;
+  width: 295px;
+  height: 87px;
+  color: #fff;
+  margin-bottom: 0;
+  padding-left: 10px;
+}
+
+.profesional-Art-ul {
+  padding-left: 10px;
+  width: 70%;
+  margin-top: 0;
+}
+
+.profesional-Art-li {
+  font-family: 'Inter', sans-serif;
+  font-weight: 500px;
+  font-size: 15px;
+  line-height: 20px;
+  color: #fff;
+  padding: 10px 15px;
+  background: rgba(255, 255, 255, 0.24);
+}
+
+a {
+  display: block;
+  display: flex;
+  justify-content: center;
+}
+
+.profesional-Art-btm {
+  width: 100%;
+  height: 48px;
+  background: #ff6b00;
+  font-size: 17px;
+  color: #fff;
+  text-align: center;
+  letter-spacing: 0.03em;
+  margin-top: 22px;
+  border-style: none;
+}
+
+a.profesional-Art-btm {
+  display: flex;
+  align-items: center;
+}
+
+/* ..........footer section.......... */
+footer {
+  width: 100%;
+  background: #1c1a19;
+}
+
+h2 {
+  color: #ff6b00;
+  font-weight: 400;
+  font-size: 72px;
+  line-height: 88px;
+  width: 327px;
+  height: 79px;
+  padding: 120px 0 0 24px;
+  font-family: 'Crete Round', sans-serif;
+}
+
+.footer-text {
+  font-family: 'Inter', sans-serif;
+  font-weight: 400;
+  font-size: 16px;
+  line-height: 24px;
+  color: #f4f5f7;
+  width: 327px;
+  height: 148px;
+  padding-left: 24px;
+}
+
+.footer-btm {
+  width: 161px;
+  padding: 12px;
+  gap: 10px;
+  margin-top: 0;
+  background: #ff6b00;
+  color: #fff;
+  margin-left: 24px;
+  border: none;
+}
+
+.footer-img {
+  padding-left: 212px;
+}
+
+.footer-hr {
+  border: 1px solid #fff;
+  width: 90%;
+  margin: auto;
+}
+
+.footer-list {
+  text-align: center;
+  padding-top: 100px;
+}
+
+.footer-heading {
+  font-size: 22px;
+  font-weight: 700px;
+}
+
+.footer-ul {
+  display: inline-block;
+}
+
+.footer-li {
+  font-family: 'Inter', sans-serif;
+  font-weight: 400;
+  font-size: 15px;
+  color: #fff;
+  padding-bottom: 32px;
+}
```

Comment:
Great job adding the list of technologies 😄! Please take into account to add the corresponding symbols to each one of the headers, as displayed below:

| Figma design  | Your design |
| ------------- | ------------- |
| ![image](https://user-images.githubusercontent.com/42799579/206874223-7f988729-1557-4758-ad40-09b5efc3a115.png)  | ![image](https://user-images.githubusercontent.com/42799579/206874226-5523ba7a-9d3f-4e22-970e-7788adf6d94d.png)  |


> Keep in mind that this project's goal is to improve your ability to follow design guidelines (imagine Microverse as a client requesting a website) 😉.